test/units/utils/test_encrypt.py

```diff
@@ -28,15 +28,26 @@ def __exit__(self, exception_type, exception_value, traceback):
 def assert_hash(expected, secret, algorithm, **settings):
 
     if encrypt.PASSLIB_AVAILABLE:
-        assert encrypt.passlib_or_crypt(secret, algorithm, **settings) == expected
+        assert encrypt.do_encrypt(secret, algorithm, **settings) == expected
         assert encrypt.PasslibHash(algorithm).hash(secret, **settings) == expected
     else:
-        assert encrypt.passlib_or_crypt(secret, algorithm, **settings) == expected
+        assert encrypt.do_encrypt(secret, algorithm, **settings) == expected
         with pytest.raises(AnsibleError) as excinfo:
             encrypt.PasslibHash(algorithm).hash(secret, **settings)
         assert excinfo.value.args[0] == "passlib must be installed and usable to hash with '%s'" % algorithm
 
 
+@pytest.mark.skipif(sys.platform.startswith('darwin'), reason='macOS requires passlib')
+def test_passlib_or_crypt():
+    # deprecated: description='passlib_or_crypt deprecated' core_version='2.20'
```

Comment:
No need to include a deprecation comment here, since the tests will be removed when the code under test (already marked with its own deprecation) is removed.